packages/widget-playground/src/store/editTools/themes.ts

```diff
@@ -0,0 +1,13 @@
+import { windows95Theme } from '@lifi/widget';
+import type { ThemeItem } from './types';
+
+export const themeItems: ThemeItem[] = [
+  {
+    id: 'windows95',
+    name: 'Windows 95',
+    theme: windows95Theme,
+    options: {
+      restrictAppearance: 'light',
```

Comment:
this setting restricts the mode/appearance in the playground. I've also added an indicator that this is restricted and message in the the Appearance control.